resolvers/__tests__/index.spec.js

```diff
@@ -94,28 +94,6 @@ describe.skip('Query resolvers', () => {
     expect(actionsIds).toEqual(targetedCampaignActionsIds)
   })
 
-  it("returns the 'UserActions' for a given user", async () => {
```

Comment:
We are not longer returning UserActions within a user?

Reply:
Yeah this endpoint changed, we now ask for a specific actions by slug. These tests were failing due the code being removed.